.github/workflows/main.yaml

```diff
@@ -11,4 +11,4 @@ jobs:
     runs-on: self-hosted
     steps:
       - name: Echo Hello
-        run: echo hostname
```

Comment:
REmoving this line @michael-anastasakis-testing

Reply:
Replying here to trigger